fe/fe-core/src/main/java/org/apache/doris/nereids/rules/expression/rules/FunctionBinder.java

```diff
@@ -188,6 +191,15 @@ public Expression visitUnboundFunction(UnboundFunction unboundFunction, Expressi
                 // so wrap COUNT with Nvl to ensure it's result is 0 instead of null to get the correct result
                 boundFunction = new Nvl(boundFunction, new BigIntLiteral(0));
             }
+            if (context.cascadesContext.currentElementAtLevel() == 1
```

Comment:
it is hard to understand, please add enough comment to explain the if condition

Reply:
done